tests/python/unittest/test_runtime_rpc.py

```diff
@@ -102,6 +102,14 @@ def remote_array_func(y):
     fremote(r_cpu)
 
 
+def test_rpc_large_array_shrink():
+    server = rpc.Server("localhost")
+    remote = rpc.connect(server.host, server.port)
+    ctx = remote.cpu(0)
+    a = tvm.nd.array(np.ones((5041, 720)).astype('float32'), ctx)
+    b = tvm.nd.array(np.ones((720, 192)).astype('float32'), ctx)
```

Comment:
can you add some commentary explaining what assert is being done here? what are we trying to ensure doesn't happen by running this code? also, does this test need to run on e.g. an RPI to reproduce?

Reply:
great pt, right now i am only copying the failure case of the large array. 

However, the error is not exactly reproducible, because it depends on the internal state of the RPC server. Atm I am only able to fix a potential bug but not sure if we could get a exact regression.

Reply:
ok, is it possible to put a brief comment in the code? there just aren't any asserts here at all so it will be hard to tell why we should keep this test around.

Reply:
OK, I have updated the testcase name it make it less confusing